widget/ui/src/components/Toast/Toast.helpers.ts

```diff
@@ -5,3 +5,6 @@ export function idGenerator() {
     return id;
   };
 }
+
+export const TOAST_EXIT_DELAY = 400; // the delay before the toast is unmounted
+export const TOAST_TRANSITION_DURATION = 400; // the duration of the toast transition
```

Comment:
I think timing doesn't match exactly with the provided video specially in terms of exist delay. Let me double check it with the design team.

Reply:
I reduced the time.